example.conf-tmpl

```diff
@@ -32,6 +32,8 @@ input {
         # outside of GCE, you will need to specify the service account's
         # JSON key file below.
         #json_key_file => "/home/erjohnso/pkey.json"
+        # or
+        #json_key_file_content => "{****}"
```

Comment:
Please put something that's plausible as JSON content, for example (if it's the minimal thing need) like:

```suggestion
        #json_key_file_content => '{"private_key_id": "xxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxx",
    "private_key": "-----BEGIN PRIVATE KEY-----\nxxxxxxxxxxx\n-----END PRIVATE KEY-----\n",}'
```

Reply:
I have updated it as per your suggestion.